frontend/src/metabase/ui/components/overlays/Tooltip/Tooltip.styled.tsx

```diff
@@ -10,6 +10,11 @@ export const getTooltipOverrides = (): MantineThemeOverride["components"] => ({
         transition: "fade",
         duration: 200,
       },
+      events: {
+        hover: true,
+        focus: true,
+        touch: true,
+      },
```

Comment:
This ensures that the Tooltip works if you're using the keyboard. I think it makes sense to enable this globally.